data/115/889/427/3/1158894273.geojson

```diff
@@ -20,23 +20,36 @@
     "mz:is_landuse_aoi":0,
     "mz:max_zoom":18.0,
     "mz:min_zoom":14.0,
-    "name:fra_x_variant":[
+    "name:eng_x_preferred":[
+        "20th Arrondissement"
+    ],
+    "name:eng_x_variant":[
+        "20th arrondissement of Paris"
+    ],
+    "name:fra_x_preferred":[
         "20e Arrondissement"
     ],
+    "name:fra_x_variant":[
+        "20e arrondissement de paris",
+        "XXe arrondissement de paris",
+        "Paris XXe"
```

Comment:
Would it be crazy to set the `"label:fra_x_preferred"` to `"Paris XXe"` to shorten what is displayed for reverse geocoding?

Reply:
Hmm.. let me poke around, I'll come back to this one...